src/index.html

```diff
@@ -11,6 +11,10 @@
       rel="stylesheet"
       href="./styles/main.scss"
```

Comment:
The stylesheet is linked as a SCSS file (`main.scss`). Ensure that this SCSS file is compiled to CSS and linked correctly, as browsers do not understand SCSS directly.